src/translations/en-US.json

```diff
@@ -195,7 +195,7 @@
         "limit": 0
     },
     "visualization.ErrorDescriptionMissingMapboxToken": {
-        "value": "An API access token is required to use Mapbox GL in Geo chart (pushpins)",
+        "value": "The Geochart configuration requires a valid API access token.",
         "comment": "Mapbox GL should not be translated",
```

Comment:
The comment is no longer necessary. Please set it to empty string.

Reply:
fixed